src/nystrum/entites.js

```diff
@@ -856,10 +856,13 @@ const Dragging = superclass => class extends superclass {
     if (!tile) return false;
     if (tile.entities.length > 0) {
       const entity = tile.entities[0];
-      if (!this.draggedEntity && entity.entityTypes.includes('DRAGGABLE')) {
```

Comment:
we still need to check if the entity has the 'DRAGGABLE' type

Reply:
Does that mean move the check for the type out from where it is now?